blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -91,12 +93,20 @@ class RunDetailsHeader extends Component {
         );
 
         const branchUrl = `${buildPipelineUrl(run.organization, pipeline.fullName)}/activity/${run.pipeline}`;
+        const labelClassName = run.pullRequest ? 'pullRequest' : '';
 
         const branchSourceDetails = (
             <div className="u-label-value" title={branchLabel + ': ' + displayName}>
-                <label>{ branchLabel }:</label>
+                <label className={labelClassName}>{ branchLabel }:</label>
                 {isMultiBranch ? (
-                    <span><Link to={ branchUrl }>{ displayName }</Link></span>
+                    <span>
+                        <Link to={ branchUrl }>{ displayName }</Link>
+                        { run.pullRequest && run.pullRequest.url &&
+                        <a title="Display the log in new window" target="_blank" href={run.pullRequest.url}>
```

Comment:
Ah hadnt actually reviewed because I was having css issues and wnated help.

And yes the url can be null. Specifically if you have PR from an old version of github-branch-source it will be null